efi-log.ks.in

```diff
@@ -0,0 +1,36 @@
+#version=DEVEL
+#test name: efi-log
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all --initlabel
+autopart
+
+keyboard --xlayouts='cz (qwerty),us (euro)' --vckeymap=us --switch=grp:alt_shift_toggle
+lang en_US.UTF-8
+timezone America/New_York --utc
+rootpw testcase
+shutdown
+
+%packages
+@core
+%end
+
+%post --nochroot
+
+@KSINCLUDE@ efi-log-check.sh
+
+check_log_util_exist
+
+%end
+
+%post
```

Comment:
The validation boilerplate could be replaced by this fragment: https://github.com/rhinstaller/kickstart-tests/blob/master/fragments/shared/validation/success_if_result_empty_standalone.ks

Reply:
Updated according to suggestion

Reply:
Inside `%post` we should use non-`_standalone` version of the fragment.

Reply:
Or just call the `_standalone` version without %post (standalone :))

Reply:
Thanks for pointing this our - fixed now.